app/views/home/index.html.slim

```diff
@@ -0,0 +1,3 @@
+div class="text-center mt-5" 
+  h2 Welcome to 
+  h1 Ruby Google Scraper
```

Comment:
Should we support localizing these sentenses?

Reply:
Fixed in 4f0475c